command_test.go

```diff
@@ -30,100 +31,145 @@ func fakeIO(i *serpent.Invocation) *ioBufs {
 	return &b
 }
 
-func TestCommand(t *testing.T) {
-	t.Parallel()
-
-	cmd := func() *serpent.Command {
-		var (
-			verbose bool
-			lower   bool
-			prefix  string
-			reqBool bool
-			reqStr  string
-		)
-		return &serpent.Command{
-			Use: "root [subcommand]",
-			Options: serpent.OptionSet{
-				serpent.Option{
-					Name:  "verbose",
-					Flag:  "verbose",
-					Value: serpent.BoolOf(&verbose),
-				},
-				serpent.Option{
-					Name:  "prefix",
-					Flag:  "prefix",
-					Value: serpent.StringOf(&prefix),
-				},
+func SampleCommand(t *testing.T) *serpent.Command {
```

Comment:
```suggestion
func sampleCommand(t *testing.T) *serpent.Command {
```

Typically exported functions in the testing package have some kind of tooling significance, e.g. Test, Benchmark, Example.